zerver/views/muting.py

```diff
@@ -18,3 +18,10 @@ def json_set_muted_topics(request, user_profile,
     # type: (HttpRequest, UserProfile, List[List[Text]]) -> HttpResponse
     do_set_muted_topics(user_profile, muted_topics)
     return json_success()
+
+@has_request_variables
+def update_muted_topic(request, user_profile, stream=REQ(),
+                       topic=REQ(), op=REQ()):
+    # type: (HttpRequest, UserProfile, str, str, str) -> HttpResponse
+    do_update_muted_topic(user_profile, stream, topic, op)
```

Comment:
this should probably do error handling if the topic wasn't already muted on the remove operation.